CHANGELOG.md

```diff
@@ -14,6 +14,13 @@ All changes to this project will be documented in this file.
 ## [0.7.4] 2025-08-02
 
 ### Added
+- **Configurable Ping Settings**: Enhanced network connectivity reliability with configurable ping keep-alive settings
```

Comment:
This needs to be added to a new [unreleased] section of the changelog and not go into an existing release.